website/.eslintrc.json

```diff
@@ -18,7 +14,8 @@
         "varsIgnorePattern": "^_",
         "caughtErrorsIgnorePattern": "^_"
       }
-    ]
+    ],
+    "@typescript-eslint/no-non-null-assertion": 0
```

Comment:
there is a comment to disable this rule in the corresponding file, is this necessary?

Reply:
I found this rule is really annoying and doesn't have many advantages so I disable it globally, but I can revert if you want.

Reply:
maybe we can make it a warning at least.